test/gui/tst_sharing/test.feature

```diff
@@ -134,3 +134,22 @@ Feature: Sharing
             | path        | /simple-folder |
             | name        | Public link    |
         And the public should not be able to download the file "lorem.txt" from the last created public link by "Alice" on the server
+
+
+    Scenario Outline: change collaborator permissions of a file & folder
+        Given user "Alice" has created on the server folder "simple-folder"
+        And user "Alice" on the server has created file "lorem.txt"
+        And user "Brian" has been created on the server with default attributes and without skeleton files
+        And user "Alice" has shared folder "simple-folder" on the server with user "Brian"
+        And user "Alice" has shared folder "lorem.txt" on the server with user "Brian"
```

Comment:
it seems to be a bit random where we put in `on the server`. What do you think @phil-davis ? How do those sentences sound for your?

Reply:
`on the server` is better at the end:
`        Given user "Alice" has created folder "simple-folder" on the server`

`        Given user "Alice" has created file "lorem.txt" on the server`

The `with/without` clauses sound OK after "on the server". So the pattern is:

```
Given user "Username" has [created|deleted|shared|...] [some resource] on the server (with some special stuff|with permission|with email address|...)
```

Reply:
@phil-davis @individual-it This sounds good. The required step for this is already in progress here https://github.com/owncloud/client/pull/8695/files#diff-821cad5db62830778c291de069b5182ff9bcdd90951faa2cec9d223e8efbcc20R294. We can make this change in another PR where all `Given` steps can be refactored.